dist/assets/d4d85ae49e726dcf637e2eacb3651afc.md

```diff
@@ -0,0 +1,212 @@
+# DC Candidate Fundraising Integrity Scorecard
```

Comment:
nah, this is supporting the score definition in the faq;
we should either refactor that so it's a component versus a link;
if we go the component approach, we can remove the use of dangerouslySetInnerHtml attribute.